extensions/VisualEditor/wikia/modules/ve/ve.track.js

```diff
@@ -175,6 +176,18 @@ require( ['wikia.tracker'], function ( tracker ) {
 			return;
 		}
 
+		if ( ve.init.wikia.getToolbarABTestVariantNumber() === 1 && data.label === 'edit-page-ready' ) {
```

Comment:
You didn't really modify the original event - you were just sending another one instead of it.
I'm just making sure that you will be able to track for instance: if we are getting more saves in case of variant 1 vs. default?